src/main/java/games/strategy/engine/framework/GameDataFileUtils.java

```diff
@@ -9,21 +9,22 @@
 
 import com.google.common.annotations.VisibleForTesting;
 
-import games.strategy.engine.ClientContext;
+import games.strategy.triplea.settings.ClientSetting;
 
 /**
  * A collection of utilities for working with game data files.
  */
 public final class GameDataFileUtils {
   private static final IOCase DEFAULT_IO_CASE = IOCase.SYSTEM;
 
-  private static final SaveGameFormat DEFAULT_SAVE_GAME_FORMAT =
-      ClientContext.gameEnginePropertyReader().useNewSaveGameFormat()
-          ? SaveGameFormat.NEW
-          : SaveGameFormat.CURRENT;
-
   private GameDataFileUtils() {}
 
+  private static SaveGameFormat getDefaultSaveGameFormat() {
```

Comment:
Looking at the helper methods, something looks to have drove a good deal of indirection. All that is apparently happening is we add an 'x' to the end of the save game name for the 'new' format, though it goes through quite a bit to get there :smile: 


*edit* reading through a bit further, it does seem more complex given the mac extensions :face_with_head_bandage: 
I was hoping this could be something as simle as :  ClientSetting.TEST_USE_NEW_SAVE_GAME_FORMAT.booleanValue() ? "tsvg" : "tsvgx", but no.. 

Regardless, there may be some simplifications here, but maybe not too (could be just that you've done a nice job of hiding a good bit of the complexity, this does smell of something more complex than it seems)